frontend/packages/metal3-plugin/src/constants.ts

```diff
@@ -34,8 +35,8 @@ export const HOST_STATUS_TITLES = {
   [HOST_STATUS_DEPROVISIONING]: 'Deprovisioning',
   [HOST_STATUS_MAKING_HOST_AVAILABLE]: 'Making host available',
   [HOST_STATUS_VALIDATION_ERROR]: 'Validation Error(s)',
-  [HOST_STATUS_REGISTRATION_ERROR]: 'Registration Error',
-  [HOST_STATUS_PROVISIONING_ERROR]: 'Provisioning Error',
+  [HOST_STATUS_REGISTRATION_ERROR]: 'Registration error',
+  [HOST_STATUS_PROVISIONING_ERROR]: 'Provisioning error',
```

Comment:
Why lower case?

Reply:
It resembles the wireframes and the convention used across the states

Reply:
OK :)